ex26.py

```diff
@@ -0,0 +1,39 @@
+game = [[1, 2, 0],
+	    [2, 1, 0],
+	    [2, 1, 1]]
+
+def get_winner(board):
+    #for rows
+    for i in range(3):
+        row = set(board[i])
+        if len(row) == 1 and board[i][0] != 0:
+            return board[i][0]
+
+    #for columns
+    for i in range(3):
+        col = list([board[0][i],board[1][i],board[2][i]])
+        column = set(col)
+        if len(column) == 1 and board[0][i] != 0:
+            return board[0][i]
+
+    #for diagonals
+
+    dia1 = list([board[0][0],board[1][1],board[2][2]])
+    dia2 = list([board[0][2],board[1][1],board[2][0]])
+    d1 = set(dia1)
```

Comment:
Why declare additional variables i.e d1 & d2 as we are not using the dia1 & dia2 anywhere else. Try to name the variables appropriately. Looking at dia1, dia2, d1 & d2 I don't get the context of the data stored inside it.